packages/extension-chakra-storefront/package.json

```diff
@@ -14,12 +14,12 @@
     "lint": "npm run lint:js",
     "lint:fix": "npm run lint:js -- --fix",
     "lint:js": "pwa-kit-dev lint \"**/*.{js,jsx,ts,tsx}\"",
-    "test": "pwa-kit-dev test",
+    "test": "pwa-kit-dev test -- --testPathIgnorePatterns='.*'",
     "typecheck": "tsc --noEmit"
   },
   "devDependencies": {
+    "@ark-ui/react": "^5.8.0",
     "@chakra-ui/react": "^3.16.1",
-    "@chakra-ui/skip-nav": "^2.0.15",
```

Comment:
there is no v3 of @chakra-ui/skip-nav at the moment. It seems to be relatively easy created components, which we can do directly in our storefront